client/types/instance-stats/common/InstanceStatsTable.FieldStats.tsx

```diff
@@ -62,59 +62,69 @@ export const InstanceStatsTableFieldStats = ({
 	return (
 		<Container>
 			<Ul>
-				{clients.map(({ clientName, clientVersions }) => (
-					<Li key={clientName}>
-						<CommonPadding>
-							<Typography variant="caption" component="h5">
-								{clientName}
-							</Typography>
-						</CommonPadding>
-						<Table component={TableComponent}>
-							<TableBody>
-								{clientVersions.map(
-									({ clientVersion, usageStats }) => {
-										const error = usageStats?.error ?? 0;
-										const success =
-											usageStats?.success ?? 0;
-										const total = error + success;
+				{clients.map(({ clientName, clientVersions }) => {
```

Comment:
It is good, but I think it needs to be done on the graphQL resolver as well to avoid sending useless data though the network 😄

Reply:
Yes you right !